x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -1002,26 +990,6 @@ query = '''
 registry where arrayContains(bytes_written_string_list, "missing", "en-US")
 '''
 
-[[queries]]
-expected_event_ids = [82]
-query = "file where serial_event_id - 1 == 81"
-
-[[queries]]
-expected_event_ids = [82]
-query = "file where serial_event_id + 1 == 83"
-
-[[queries]]
-expected_event_ids = [82]
-query = "file where serial_event_id * 2 == 164"
-
-[[queries]]
-expected_event_ids = [82]
-query = "file where serial_event_id / 2 == 41"
-
-[[queries]]
-expected_event_ids = [82]
-query = "file where serial_event_id % 40 == 2"
```

Comment:
Can we have more complex math operations?
Few ideas:
- put the constant first and the field name second
- use the minus sign for the field as well: `file where -serial_event_id + 100 == 18`
- have a more complex arithmetic set of operations: `file where serial_event_id + ((1 + 3) * 2 / (3 - 1)) * 2 == 90 OR 70 + serial_event_id < 100`

Reply:
Added all of the above.